Traceback (most recent call last):
  File "preprocess.py", line 29, in <module>
    f = open(fileName, "r")
IOError: [Errno 2] No such file or directory: '/home/thomas/euler2-demos2/euler2-demos2/input.txt'
Traceback (most recent call last):
  File "/home/thomas/EulerX/src-el/euler", line 92, in <module>
    main()
  File "/home/thomas/EulerX/src-el/euler", line 85, in main
    elRunner.run(args)
  File "/home/thomas/EulerX/src-el/runner.py", line 39, in run
    taxMap = TaxonomyMapping(args)
  File "/home/thomas/EulerX/src-el/alignment.py", line 149, in __init__
    os.path.join(self.inputfilesdir, self.name+".txt"))
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: '/home/thomas/euler2-demos2/euler2-demos2/input.txt'
Traceback (most recent call last):
  File "lattice.py", line 71, in <module>
    f = open(fileName, "r")
IOError: [Errno 2] No such file or directory: '/home/thomas/euler2-demos2/euler2-demos2/input.txt'
